tools/util/marshaller.go

```diff
@@ -87,7 +85,6 @@ func MarshallObject(obj interface{}, writer io.Writer) error {
 	// remove "managed by operator" label...
 	labels, exists, err := unstructured.NestedMap(r.Object, "metadata", "labels")
 	if exists {
-		delete(labels, v1.ManagedByLabel)
```

Comment:
I don't understand this change

Reply:
Its pulling in kubevirt for no apparent reason it should have never had this label.

Reply:
I see. Seems like the rest of the code block can go, too, No need to get the labels and set them back if we make no changes to the labels.

Reply:
You make a good point, there is no reason for that either. I found it because I was bumping versions, and that pulling kubevirt was causing all kinds of issues.